internal/pkg/cli/task_run.go

```diff
@@ -231,7 +233,7 @@ func (o *runTaskOpts) Validate() error {
 	}
 
 	if o.image != "" && o.dockerfilePath != "" {
-		return errors.New("cannot specify both image and Dockerfile path")
+		return errors.New("specified both image and Dockerfile path")
```

Comment:
lol this is not a wrapped error i think we should keep it as it is now. Otherwise users wouldn't know why it is an error as `specified both image and Dockerfile path`